README.md

```diff
@@ -60,6 +60,15 @@ make test
 make schema
 ```
 
+### Generating proto files
+
+Neutron proto files represented as generated Rust code is a part of the Neutron SDK. In case Neutron
+proto files have changed there's a command for Rust generated code rebuild. To rebuild the files,
+run the following command using the respective Neutron revision (commit hash) as an argument, e.g.:
+```sh
+make build-proto revision=60b7d38358efb2f8e45e25ee4db8e19a6462dd9e
```

Comment:
Btw `make build-proto revision=v2.0.3` (tags) also works, so maybe it's worth so simplify readme example with a tag, not a commit

Reply:
Cause in normal circumstances we should use tag anyway, not just a random commit